docs/graph/PowerBIEmbedded.md

```diff
@@ -41,10 +41,15 @@ The [PowerBIEmbedded Control](https://docs.microsoft.com/dotnet/api/microsoft.to
      - Read all users' basic profiles
```

Comment:
This list is rendering a bit weird in the sample app:

![image](https://user-images.githubusercontent.com/24302614/42787521-b0a2f84a-890f-11e8-8e20-d5a582b00a8c.png)

Reply:
@michael-hawker  That appears the toolkit is using the different markdown standards from GitHub, I just did some workaround so it should look good now.